docs/resources/Message.md

```diff
@@ -381,7 +381,7 @@ The encoding, and the waveform details, are an implementation detail and may cha
 | message\* | partial [message](#DOCS_RESOURCES_MESSAGE/message-object) object | minimal subset of fields in the forwarded message |
 
 \* The current subset of message fields consists of:
-`type`, `content`, `embeds`, `attachments`, `timestamp`, `edited_timestamp`, `flags`, `mentions`, `mention_roles`.
+`type`, `content`, `embeds`, `attachments`, `timestamp`, `edited_timestamp`, `flags`, `mentions`, `mention_roles`, `stickers`, `sticker_items`, and `components`.
```

Comment:
Its necesary mention stickers based in how that field is supposed to be deprecated?